glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Multimedia 
++ #### *Packet* is a formatted unit of data transmitted over network 
```

Comment:
I guess it would be good to give some examples here, like a link to an RTP packet structure scheme.
As well it might be worthed to describe the why we need fragmentation.